### ISSUE TITLE
Add "iconUrl" for remote icon files

### ISSUE THREAD

**Issue by jashsayani** [ParsePlatform/parse-dashboard#526]
I see that the dashboard allows for `iconsFolder` and `iconName` for each app. This is a decent way of pointing to an icons directory and defining icons for each app, but I would like to just point to an `iconUrl` that is a static file hosted somewhere (essentially linking to AppStore or Google Play CDN of the app icon asset). This way, when the icon is updated for the app, the dashboard would show the new icon (or break if CDN link changes on update), and I can update the link with the new icon.

**Comment by natanrolnik** [comment]
@jashsayani Yep, that's an interesting improvement to the currently existing feature. I implemented, with the help of Drew Gross, the support only for icons in the `iconsFolder`. PRs are welcome.

**Comment by markmssd** [comment]
That would be very interesting indeed!
